server/routers/api-router.js

```diff
@@ -55,16 +55,17 @@ router.get("/api/push/:pushToken", async (request, response) => {
         let duration = 0;
 
         let bean = R.dispense("heartbeat");
-        bean.time = R.isoDateTime(dayjs.utc());
+        bean.time = R.isoDateTimeMillis(dayjs.utc());
 
         if (previousHeartbeat) {
             isFirstBeat = false;
             previousStatus = previousHeartbeat.status;
             duration = dayjs(bean.time).diff(dayjs(previousHeartbeat.time), "second");
         }
 
-        log.debug("router", "PreviousStatus: " + previousStatus);
-        log.debug("router", "Current Status: " + status);
+        log.debug(`/api/push/ called at ${dayjs().format("YYYY-MM-DD HH:mm:ss.SSS")}`);
+        log.debug("PreviousStatus: " + previousStatus);
+        log.debug("Current Status: " + status);
```

Comment:
Whi did you removed first parameter?

Reply:
Looks like that first parameter was added after I first submitted the PR, so I probably missed the change when I did my rebase merge!